RiotSharp/Cache.cs

```diff
@@ -266,7 +265,12 @@ public int Count()
 
         private void StartObserving<K>(K key, TimeSpan timeSpan)
         {
-            Observable.Timer(timeSpan).Subscribe(x => TryPurgeItem(key));
+            Timer timer = null;
+            timer = new Timer(x =>
+            {
+                TryPurgeItem(key);
+                timer?.Dispose();
+            }, key, timeSpan, TimeSpan.Zero);
```

Comment:
Can you replace `TimeSpan.Zero` with `TimeSpan.FromMilliseconds(-1)` to disable periodic signaling. I know that it's technically not needed because of the `timer?.Dispose()`, but that way it is cleaner.